benchmark/benchmark.go

```diff
@@ -232,39 +232,45 @@ func DoUnaryCall(tc testgrpc.BenchmarkServiceClient, reqSize, respSize int) erro
 }
 
 // DoStreamingRoundTrip performs a round trip for a single streaming rpc.
-func DoStreamingRoundTrip(stream testgrpc.BenchmarkService_StreamingCallClient, reqSize, respSize int) error {
+func DoStreamingRoundTrip(
+	stream testgrpc.BenchmarkService_StreamingCallClient, reqSize, respSize, streamCount int) error {
 	pl := NewPayload(testpb.PayloadType_COMPRESSABLE, reqSize)
 	req := &testpb.SimpleRequest{
 		ResponseType: pl.Type,
 		ResponseSize: int32(respSize),
 		Payload:      pl,
 	}
-	if err := stream.Send(req); err != nil {
-		return fmt.Errorf("/BenchmarkService/StreamingCall.Send(_) = %v, want <nil>", err)
-	}
-	if _, err := stream.Recv(); err != nil {
-		// EOF is a valid error here.
-		if err == io.EOF {
-			return nil
+	for i := 0; i < streamCount; i++ {
```

Comment:
Should these streams happen in parallel instead? Something like:

```golang
var wg sync.WaitGroup
wg.Add(streamCount)
for i := 0; i < streamCount; i++ {
	go func() {
		defer wg.Done()
		if err := stream.Send(req); err != nil {
			return fmt.Errorf("/BenchmarkService/StreamingCall.Send(_) = %v, want <nil>", err)
		}
		if _, err := stream.Recv(); err != nil {
			// EOF is a valid error here.
			if err == io.EOF {
				return nil
			}
			return fmt.Errorf("/BenchmarkService/StreamingCall.Recv(_) = %v, want <nil>", err)
		}
	}()
}
wg.Wait()
```

Reply:
Done.
I also do this in the `DoByteBufStreamingRoundTrip`.